ds-prep-python-conditionals/main.py

```diff
@@ -0,0 +1,53 @@
+day_of_week = 'Monday'
+if day_of_week == 'Saturday' or day_of_week == 'Sunday':
+    print('It is the weekend')
+else:
+    print('It is a weekday')
+
+
+student_1_score = 50
+if student_1_score >= 70:
+    pass_or_fail = 'You passed!'
+else:
+    pass_or_fail = 'You failed!'
+
+print('Student_1_score:', student_1_score, 'Pass_or_fail:', pass_or_fail)
+
+student_2_score = 50
+if student_2_score < 60:
+  letter_grade = 'F'
+elif student_2_score >= 60 and student_2_score < 70:
+    letter_grade = 'D'
+elif student_2_score >= 70 and student_2_score < 80:
+    letter_grade = 'C'
+elif student_2_score >= 80 and student_2_score < 90:
+    letter_grade = 'B'
+elif student_2_score >= 90 and student_2_score <= 100:
+    letter_grade = 'A'
+elif student_2_score >= 100:
+    letter_grade = 'A+'
```

Comment:
`letter_grade` should be `'A+'` if `student_2_score` is `100`, not `'A'`.

---

![image](https://user-images.githubusercontent.com/16143771/148879267-f043b104-6ee3-48fb-ae6e-dbc44bdfefa6.png)

Reply:
Fixed the Logic , changed <= to < condition